docs/fundamentals/flow/create-flow.md

```diff
@@ -266,6 +266,11 @@ Based on your local Jina version, Jina Hub may rebuild the Docker image during t
 If you do not wish to rebuild the image, set the environment variable `JINA_HUB_NO_IMAGE_REBUILD`.
 ```
 
+```{tip}
+If an Executor requires volumes to be mapped for them to persist data, Jina will create a StateFulSet for it instead of a Deployment.
```

Comment:
```suggestion
If an Executor requires volumes to be mapped for them to persist data, Jina will create a StatefulSet for that Executor instead of a Deployment.
```